scripts/build/clone_guides.rb

```diff
@@ -22,15 +22,16 @@
 # For non-production sites, select the correct branch of the guide
 # --------------------------------------------
 # For the interactive guides, only build the dev branch for non-prod sites
-guide_branch = 'prod'
-fallback_guide_branch = 'master'
+# Note:  temporarily changing the guide_branch to gradle on demo1 for testing purposes.
+guide_branch = 'gradle'
```

Comment:
This looks to be a local development only change. Should these lines be removed before from this pull request?